frappe/custom/doctype/customize_form/customize_form.js

```diff
@@ -88,17 +90,11 @@ frappe.ui.form.on("Customize Form", {
 	},
 
 	setup_sortable: function(frm) {
-		frm.page.body.find(".highlight").removeClass("highlight");
 		frm.doc.fields.forEach(function(f, i) {
-			var data_row = frm.page.body.find(
-				'[data-fieldname="fields"] [data-idx="' + f.idx + '"] .data-row'
-			);
-
-			if (f.is_custom_field) {
-				data_row.addClass("highlight");
-			} else {
+			if (!f.is_custom_field) {
```

Comment:
Perhaps `sortable` is also something we should be setting in `grid-row-render`? I'm not sure if `sortable=false` works on the second page.

Reply:
If we do this, then the function name will also need to be changed I guess 😅 

Sorry meri batti phases mein jal rahi hai 🤦🏼

Reply:
It works since we are setting `f._sortable == false` for all the non-custom fields not only fields that are on page 1.
adding class was not working because the rows in other pages were not actually present in the dom.